inference-engine/ie_bridges/python/src/openvino/offline_api/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+# Copyright (C) 2018-2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set(TARGET_NAME "offline_api")
+
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PYTHON_BRIDGE_OUTPUT_DIRECTORY}/offline_api)
+
+file(GLOB SOURCE
+        ${CMAKE_CURRENT_SOURCE_DIR}/offline_api.pyx
+        ${CMAKE_CURRENT_SOURCE_DIR}/offline_api_impl.cpp)
+
+set_source_files_properties(${SOURCE} PROPERTIES CYTHON_IS_CXX ON)
+
+# create target
+
+cython_add_module(${TARGET_NAME} ${SOURCE})
+set(INSTALLED_TARGETS ${TARGET_NAME})
+add_dependencies(${TARGET_NAME} ie_api)
+
+# build offline_api module
```

Comment:
Do you have a plan to remove commented code?

Reply:
removed.